socorro/unittest/scripts/test_add_crashid_to_queue.py

```diff
@@ -0,0 +1,67 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+
+from mock import MagicMock, patch
+import pytest
+
+from socorro.lib.ooid import create_new_ooid
+from socorro.scripts.add_crashid_to_queue import (
+    main,
+)
+
+
+def test_missing_args(capsys):
+    # Make sure that running main with no args causes the script to exit with exit_code 2
+    with pytest.raises(SystemExit) as exc_info:
+        main([])
+    assert exc_info.exconly() == 'SystemExit: 2'
```

Comment:
This relies on the repr of the SystemExit exception which shouldn't change ever plus the check seemed pretty readable. Having said that, I'm not wildly excited about comparing against the repr of an exception in general.